### ISSUE TITLE
Generated doc is empty

### ISSUE THREAD

**Issue by SeNeO** [tomaz/appledoc#561]
I'm using the following script to generate documentation on a test project 
>~/Documents/appledoc/appledoc \
>--project-name "Test" \
>--project-company "Test" \
>--company-id "com.test" \
>--output "~/Desktop/appledoctest/" \
>--logformat xcode \
>--merge-categories \
>--no-create-docset \
>"./testAppleDoc/"

XCode build the target without issue and the files are generated but they are empty, only the static elements are present (index/hierarchy links etc...).
I don't know what I'm doing wrong, there is no warning (so I think I'm properly documenting my code), no errors.

As anyone had this kind of issue?

**Comment by TongKuo** [comment]
Yep, I have the same issue. I bypass this issue by using [appledoc 2.2 (build 963)](https://github.com/tomaz/appledoc/releases/tag/v2.2-963) which can work properly. But all the later versions don't work for me.

**Comment by SeNeO** [comment]
Thanks for the answer, It indeed works with appledoc 2.2 (build 963).

**Comment by tomaz** [comment]
Try increasing verbosity with `--verbose 4` (or greater) to see if there's some meaningful info in log.

**Comment by aaronpk** [comment]
Thanks, I had to build my docs using appledoc 2.2 build 963 rather than the version that installed using Homebrew.

**Comment by dreid93** [comment]
I have also experienced this problem using the following build script : 

```
company="AdLyft Inc";
companyID="com.AdLyft";
companyURL="http://adlyft.com";
target="iphoneos";
outputPath="${PROJECT_DIR}/Documentation";

/usr/local/bin/appledoc \
--project-name "${PROJECT_NAME}" \
--project-company "${company}" \
--company-id "${companyID}" \
--docset-atom-filename "${company}.atom" \
--docset-feed-url "${companyURL}/${company}/%DOCSETATOMFILENAME" \
--docset-package-url "${companyURL}/${company}/%DOCSETPACKAGEFILENAME" \
--docset-fallback-url "${companyURL}/${company}" \
--output "${outputPath}" \
--publish-docset \
--docset-platform-family "${target}" \
--logformat xcode \
--keep-intermediate-files \
--no-repeat-first-par \
--no-warn-invalid-crossref \
--no-create-docset \
--exit-threshold 2 \
"${PROJECT_DIR}/AdLyft/ADLAdLyftController.h" \
"${PROJECT_DIR}/AdLyft/Models/ADLReward.h" \
"${PROJECT_DIR}/AdLyft/Models/ADLResult.h"
```

**Comment by TorreyBetts** [comment]
This happened to me several months ago. In my case, I fixed the issue through the following steps.

 1. Delete the **~/.appledoc** folder
 2. Clone the appledoc repository: **git clone git://github.com/tomaz/appledoc.git**
 3. Add the template option for the installation command: **sudo sh install-appledoc.sh -t default**

**Comment by jameswangz** [comment]
This works for me,  thanks @TorreyBetts!

**Comment by thomasmso** [comment]
Still working today! Thanks @TorreyBetts!

**Comment by tmcguire** [comment]
The installation method described in https://github.com/tomaz/appledoc/issues/561#issuecomment-149327949 worked for me as well. 

Installing via homebrew did not work. And indeed the README says that installing via homebrew does not install templates. Maybe the homebrew package should include templates to make it just work?
